Secret_Of_Console/Secret_Of_Console/Game.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  Game.swift
+//  Secret_Of_Console
+//
+//  Created by axel leydier on 25/01/2019.
+//  Copyright © 2019 axelleydier. All rights reserved.
+//
+
+import Foundation
+
+class Game {
+    
+    var players: [Player] = []
+    
+    func start() {
+        print("Le jeu a démarré")
+        settings()
+        play()
+        end()
+    }
+    
+    func settings() {
+        // Répéter 2x:
+        //      Choix du nom de l'équipe
+        //      Répéter 3x:
+        //              Choix du type de perso
+        //              Choix du nom du personnage
+        // On démarre le jeu on appelle la fonction play()
+        
+        for playerIndex in 1...2 {
+            
+            print("Joueur \(playerIndex): Merci de rentrer votre nom: ")
+            
+            var name = ""
+            var nameCounter = 0
+            repeat {
+                if nameCounter >= 1 {
+                    print("Merci de rentrer un nom NON VIDE")
+                }
+                if let _name = readLine() {
+                    name = _name
+                }
+                nameCounter += 1
+            } while name.isEmpty
+            
+            let player = Player(name: name)
+            
+            for characterIndex in 1...3 {
+                
+                if characterIndex == 3
```

Comment:
Instead of adding some `if-else` you should use the previous way, by creating a counter, increase it and look for a specific value -> see line 37.